packages/page-alerts/src/js/react.js

```diff
@@ -0,0 +1,84 @@
+/*! [replace-name] v[replace-version] */
+/***************************************************************************************************************************************************************
+ *
+ * Page alert function
+ *
+ * Use page alert to notify and alert users of important application events.
+ *
+ **************************************************************************************************************************************************************/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+
+/**
+ * Page alert info
+ *
+ * @param  {string} children - Anything inside the component
+ */
+export const PageAlertInfo = ( { children } ) => {
```

Comment:
as above comments: `<PageAlert as="..." />` seems to reduce API surface area here as well.